Documentation/kubernetes.md

```diff
@@ -41,4 +41,4 @@ Kubernetes 1.6 requires CNI plugin version 0.5.1 or later.
 
 See [troubleshooting](troubleshooting.md)
 
-[kubeadm]: https://kubernetes.io/docs/getting-started-guides/kubeadm/
+[kubeadm]: https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network
```

Comment:
Do we want to jump to the `#pod-network` header?

Reply:
Thanks @Oats87 for comment!

This link is used under this context
> For information on deploying flannel manually, using the Kubernetes installer toolkit kubeadm, see Installing Kubernetes on Linux with kubeadm.

So I think it's better to point to page/header explaining how to deploy flannel with kubeadm.

Reply:
I'm not so sure I agree with that. The text that is referencing the `kubeadm` link is:
>Installing Kubernetes on Linux with kubeadm

which would lead me to believe that the link will take me to the overall Installing instructions, rather than specifically the "after you've installed K8s here is how you install a pod network"

Reply:
Thanks @Oats87 for comment.
OK. I follow your advice.